### PR TITLE
add error block

### PR DESCRIPTION
**Pull request by jordan621k**

### CODE REVIEW

File: src/component/Form.js [hunk 1]
```diff
@@ -31,12 +32,16 @@ class Form extends React.Component {
         window.alert(response.result.error.message)
       } else {
         const sortedVideos = response.result.items.sort((a, b) => (parseInt(a.statistics.viewCount) < parseInt(b.statistics.viewCount)) ? 1 : -1)
+        this.setState({ showError: false })
```
Comment (x955199) [inline src/component/Form.js]: why? what logic?

File: src/component/Form.js [hunk 2]
```diff
@@ -31,12 +32,16 @@ class Form extends React.Component {
...
         videoList.set(sortedVideos)
       }
-      isLoading.set(false)
     } catch (error) {
-      console.log(error)
+      console.log(error.body)
+      this.errorMessage = error.body
+      videoList.set([])
```
Comment (x955199) [inline src/component/Form.js]: why?

File: src/component/Form.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { videoList, isLoading } from '../store/VideoStore'
 class Form extends React.Component {
   constructor (props) {
     super(props)
+    this.state = { showError: false }
```
Comment (x955199) [inline src/component/Form.js]: hasError better

File: src/component/Form.js [hunk 4]
```diff
@@ -132,6 +137,9 @@ class Form extends React.Component {
           <br/>
           {/* <input type="submit" value="Submit"/> */}
         </form>
+        {this.state.showError && <div className="Errorbox">
```
Comment (x955199) [inline src/component/Form.js]: Error

File: src/component/Form.js (hunk 1 above)
Comment (jordan621k) [reply src/component/Form.js]: if the logic shown first it has to be turn off if back to normal

File: src/component/Form.js (hunk 2 above)
Comment (jordan621k) [reply src/component/Form.js]: why what

Comment (jordan621k) [reply src/component/Form.js]: or the videoList remain whatever the last time was

**Pull request merged**